### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by jyash97**

### CODE REVIEW

File: components/SearchSandbox/containers/Editor.js [hunk 1]
```diff
@@ -425,7 +425,12 @@ export default class Editor extends Component {
 	);
 
 	renderDeleteJSON = res => (
-		<Popconfirm title="Are you sure to delete this JSON?" placement="bottomRight" onConfirm={() => this.handleDeleteJSON(res._id)} okText="Yes">
+		<Popconfirm
+			title="Are you sure to delete this JSON?"
```
Comment (siddharthlatest) [inline components/SearchSandbox/containers/Editor.js]: @jyash97 This isn't a proper phrase.

Comment (metagrover) [reply components/SearchSandbox/containers/Editor.js]: @jyash97 can you update this?

Comment (jyash97) [reply components/SearchSandbox/containers/Editor.js]: Yes updating

File: components/SearchSandbox/containers/Editor.js [hunk 2]
```diff
@@ -486,27 +491,53 @@ export default class Editor extends Component {
 			sortBy: 'asc',
 			paginationAt: 'bottom',
 			...resultComponentProps,
+			renderSuggestions: ({
```
Comment (metagrover) [inline components/SearchSandbox/containers/Editor.js]: @jyash97 Can you explain what this is? `renderSuggestion` is a prop on DataSearch. Refer docs here: https://opensource.appbase.io/reactive-manual/search-components/datasearch.html

**Comment by metagrover** [comment]
This is non-operational. Please test and update the PR to include the fix.

**Comment by divyanshu013** [comment]
Can this be merged now?

**Comment by jyash97** [comment]
Closed in favor of #51

**Pull request closed without merging**